config/master.d/master.conf

```diff
@@ -3,7 +3,8 @@ auto_accept: True
 interface: 0.0.0.0
 event_return: mysql
 presence_events: True
-worker_threads: 10
+worker_threads: 16
+timeout: 180
```

Comment:
@ereslibre : As you said 5 seconds for this option is way to low. Problem which I'm trying to fix is on virtualized environment. It means that we have to expect much longer times of executing commands and operations than on bare metal. Problem with etcd is related to ETCD_ELECTION_TIMEOUT which is also quite low default only 100ms. https://coreos.com/etcd/docs/latest/tuning.html